cpu/samd5x/cpu.c

```diff
@@ -64,7 +64,13 @@ static void dfll_init(void)
 #endif
     ;
 
-    OSCCTRL->DFLLCTRLB.reg = reg;
+    /* workaround for Errata 2.8.3 DFLLVAL.FINE Value When DFLL48M Re-enabled */
+    OSCCTRL->DFLLMUL.reg = 0;   /* Write new DFLLMULL configuration */
+    OSCCTRL->DFLLCTRLB.reg = 0; /* Select Open loop configuration */
+    OSCCTRL->DFLLCTRLA.bit.ENABLE = 1; /* Enable DFLL */
+    OSCCTRL->DFLLVAL.reg = OSCCTRL->DFLLVAL.reg; /* Reload DFLLVAL register */
+    OSCCTRL->DFLLCTRLB.reg = reg; /* Write final DFLL configuration */
+
```

Comment:
I think this is unrelated to this PR and should be move in another PR.

Reply:
Ok, dropped it.